packages/dds/map/package.json

```diff
@@ -23,7 +23,10 @@
     "build:full": "npm run build",
     "build:full:compile": "npm run build:compile",
     "build:genver": "gen-version",
-    "build:test": "tsc --project ./src/test/tsconfig.json",
+    "build:test": "concurrently npm:build:test:mocha npm:build:test:memory npm:build:test:types",
```

Comment:
why split the build apart?

Reply:
i just want to ensure we have a need for this, as i find having more tasks slows down the build, as each task spawns a process which can be costly. compiling them all at once in a single tsc process will generally be faster.

Reply:
No real reason tbh, just following the pattern I saw in other places where we have several sets of tests. I'll check about keeping a single tsconfig file here and building them all at once.